README.md

```diff
@@ -18,8 +18,20 @@
 
 The preview button is wired up to the `ServiceName` field first, then `REST`. If there is no value in either field the button is hidden.
 
-### Custom Category URL Examples
+### Optional URL Parameters
+
+A URL may contain only one of `cat` or `catGroup`. If both are present, `cat` will be used. The other parameters can be mixed and matched as needed.
+
+`cat` and `catGroup` are mutually exclusive
+
+`products` accepts a zero-based index of the product checkboxes that you would like to be checked by default. You may pass multiple values separated by commas.
```

Comment:
**question**: will all new data be added to the end of the index and if data is ever removed will the indexes stay consistent?

Reply:
These have been hard-coded for years. I don't think that they will change anytime soon. Perhaps when the app is rewritten but I think that we could handle that as well. This is a good point though. However, I think at this point, I'm good with it as is.